backend/FourthPharos.Domain/CandelaObscuraCharacter/Features/CharacterRoleFeature.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FourthPharos.Domain.CandelaObscuraCharacter.Models;
+using FourthPharos.Domain.Features;
+using FourthPharos.Domain.Models;
+
+namespace FourthPharos.Domain.CandelaObscuraCharacter.Features;
+
+public sealed record CharacterRoleFeature(Character Target, CharacterSpecialty Specialty) : FeatureBase<Character>(Target)
+{
+    public override string Code => "char_role";
+
+    public CharacterRole Role { get; } = Specialty switch
```

Comment:
I don't think this gets updated when you do `feature with { Specialty = newSpecialty }`. Would be better to make it into `public CharacterRole Role => Specialty switch [...]`.